package.json

```diff
@@ -90,11 +90,12 @@
         "axios": "^1.7.7",
         "classnames": "^2.2.5",
         "core-js": "^2.5.5",
-        "firebase": "^7.24.0",
+        "firebase": "^9.23.0",
         "history": "^4.7.2",
         "lodash": "^4.17.20",
         "papaparse": "^5.4.1",
         "plotly.js": "^2.35.2",
+        "qs": "^6.14.0",
```

Comment:
I got an error that it couldn't find this but I don't know why it was working before